backend/sozisel/lib/sozisel_web/schema/resolvers/launched_event_resolvers.ex

```diff
@@ -0,0 +1,30 @@
+defmodule SoziselWeb.Schema.Resolvers.LaunchedEventResolvers do
+  alias Sozisel.Model.{LaunchedEvents, EventResults}
+  alias LaunchedEvents.LaunchedEvent
+
+  import SoziselWeb.Schema.Middleware.ResourceAuthorization, only: [fetch_resource!: 2]
+
+  def get_quiz_summary(_parent, _args, ctx) do
+    quiz_summary =
+      fetch_resource!(ctx, LaunchedEvent)
+      |> EventResults.quiz_summary()
```

Comment:
You don't have to keep the quiz_summary separate, just do `|> then(&{:ok, &1})` after the `quiz_summary()`